src/modules/ProductLicense.rb

```diff
@@ -1182,19 +1183,18 @@ def SearchForLicense_NormalRunBaseProduct(_src_id, fallback_dir)
     def SearchForLicense_AddOnProduct(src_id, _fallback_dir)
       Builtins.y2milestone("Getting license beta from repository %1", src_id)
 
+      # use a separate license directory for each product
+      @tmpdir = File.join(SCR.Read(path(".target.tmpdir")), "product-license", src_id.to_s)
+
+      # try reading the product license from libzypp first
+      return if product_license(src_id, @tmpdir)
```

Comment:
it will skip reading `@beta_file`, is it ok?

Reply:
Yes, that's OK. The Beta file cannot be placed into RPM-MD repositories, it's not accessible via libzypp from there.